controllers/dscinitialization/utils.go

```diff
@@ -304,7 +304,7 @@ func CompareNotebookNetworkPolicies(np1 netv1.NetworkPolicy, np2 netv1.NetworkPo
 
 func (r *DSCInitializationReconciler) waitForManagedSecret(ctx context.Context, name, namespace string) (*corev1.Secret, error) {
 	managedSecret := &corev1.Secret{}
-	err := wait.PollUntilContextTimeout(ctx, resourceInterval, resourceTimeout, false, func(ctx context.Context) (done bool, err error) {
+	err := wait.PollUntilContextTimeout(ctx, resourceInterval, resourceTimeout, false, func(ctx context.Context) (done bool, err error) { //nolint:nonamedreturns,lll // We want to capture all errors.
```

Comment:
I dont think we need named params here. Not to mention that we do not use `done` in the code at all. Maybe we just remove it?